source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.md

```diff
@@ -81,9 +81,7 @@ parse error and MUST NOT do DNS resolution or contact hosts.
 It is an error to specify more than one host name in a connection string with the `mongodb+srv` protocol, and the driver
 MUST raise a parse error and MUST NOT do DNS resolution or contact hosts.
 
-A driver MUST verify that in addition to the `{hostname}`, the `{domainname}` consists of at least two parts: the domain
-name, and a TLD. Drivers MUST raise an error and MUST NOT contact the DNS server to obtain SRV (or TXT records) if the
-full URI does not consist of at least three parts.
+In the case that an hostname does not have a `{hostname}` and a `{domainname}` (consisting of a domain name and a TLD) prior to DNS resolution, drivers MUST NOT throw an error. 
```

Comment:
Can you update the change log at the end of the file? 

Saying we "must not" do X feels natural since we're implementing this change now, but reading this in the future where we don't necessarily have the historical context it is something that was never asked of the implementer so they weren't planning on it. Maybe it would be better to frame this in the positive of what should be allowed?

Reply:
Yep, updated the change-log! I changed it to use different wording also.